assets/js/services/adverts.service.js

```diff
@@ -21,16 +21,27 @@ function Adverts ($http, $rootScope) {
   }
 
   function fetch () {
-    let advertsFeed = $rootScope.rootURL + 'adverts_feed?wallet_version=3';
+    if (!$rootScope.apiDomain.endsWith('blockchain.info/')) {
```

Comment:
Dev and staging need to have ads too, otherwise we can't test this for layout and bug issues. So I think this check can go away.